moqu/core/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.openapi.generator</groupId>
+        <artifactId>quarkus-openapi-generator-moqu-parent</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-openapi-generator-moqu-core</artifactId>
+    <name>Quarkus - Openapi Generator - Moqu - Core</name>
```

Comment:
nit - can you rename it to match the ones we do in the client?  `Quarkus :: OpenAPI Generator :: Moqu :: Core`.

Reply:
I think we are using `-` instead `::`, no?